README.md

```diff
@@ -1,4 +1,4 @@
-# SqlFormatter
+# Doctrine SqlFormatter
```

Comment:
This is not a CS change and I'm not convinced we should apply this change at all, tbh. At least not without a proper discussion.

Reply:
At least the major Doctrine projects like DBAL have it there - https://github.com/doctrine/dbal - and IMO it is better for branding as users trust Doctrine code.

Reply:
It wasn't Doctrine code from the start. I don't see the necessity for this change.

Reply:
I removed this change from this PR...